### ISSUE TITLE
Do something when clicking on parameterized tests in the test GUI

### ISSUE THREAD

**Issue by stagg54** [astemes/astemes-lunit-parameterized-test#2]
Right now I can double click on the Test VI in this case - test Consersion and it opens the test vi.

Double clicking on one of the parameterized test cases? (I don't know what the correct terminology for that is - but in this case something like 0 F to C boiling) does nothing. 

I feel like it should do something. If for no other reason than just to be consistent.

I would propose it should take you to the Parameters.vi That would enhance the usefulness of displaying the index. The index tells you which index of the array is causing problems and double-clicking on it takes you right to the array.

![image](https://github.com/user-attachments/assets/1ab93da0-80d3-4a1a-b8b2-2d596945db27)

Not sure how difficult that would be to implement. It might require changes to the internals of lunit itself. I haven't looked into how that is handled. Happy to help throw something together if you want. Just let me know ( and maybe point me in the right direction) and I'll look into opening a pull request.

**Comment by antsundq** [comment]
Yes, as we discussed I agree with you here. If it was easy to implement I would definitely have done it already.

To give some background, the problem is how LUnit UI is implemented. Currently when tests are enumerated, they are listed using their namespace as "library.lvlib:testcase.lvclass:test method.vi" and this then is translated to separate layers in the three. This is why it works eemlessly with nesting libraries and when I implemented the paramterized test, it just appends a "virtual test method" to the end, e.g. "test method.vi:parameter 1", which allowed this addon to work without making any changes to LUnit.

The problem is that when you double click on a VI in the tree control, it is opened using the qualified name, and then the "virtual test method" does not make any sense and does nothing.

Let me know if you have some ideas around this. I will also give it some thought and try to do some spiking when I find the time.
